lib/librdkafka-1.9.2/src/rdkafka_ssl.c

```diff
@@ -39,8 +39,8 @@
 #ifdef _WIN32
 #include <wincrypt.h>
 #pragma comment(lib, "crypt32.lib")
-#pragma comment(lib, "libcrypto.lib")
-#pragma comment(lib, "libssl.lib")
+#pragma comment(lib, "libcrypto64MT.lib")
+#pragma comment(lib, "libssl64MT.lib")
```

Comment:
Could you determine whether the using 64MT or 32MT suffixed libraries with CMakeLists.txt?
This change should be working only for 64bit of Windows.